framework/src/utils/MooseUtils.C

```diff
@@ -809,6 +810,20 @@ template std::string join<std::vector<MooseEnumItem>>(const std::vector<MooseEnu
 template std::string join<std::set<MooseEnumItem>>(const std::set<MooseEnumItem> &,
                                                    const std::string &);
 
+void
+createSymlink(const std::string & target, const std::string & link)
+{
+  clearSymlink(link);
+  symlink(target.c_str(), link.c_str());
```

Comment:
Check return code (This is I/O and it will fail on somebody's system at some point).